### PR TITLE
feat: v0.0.12

### PR DESCRIPTION
**Pull request by mikejgray**
# Description
Work around some issues we're seeing with intent registration
Support for Assist passthrough when PHAL plugin supports
Better logging and spoken errors

Breaking change: must use the version of the PHAL plugin [found here](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-homeassistant/pull/18) to ensure compatibility with the messagebus Message spec. This was not set correctly before but has recently been enforced in ovos-bus-client.

# Issues
#20 

# Other Notes
N/A

### CODE REVIEW

File: neon_homeassistant_skill/__init__.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
+# pylint: disable=missing-function-docstring,missing-class-docstring,missing-module-docstring
 import asyncio
 from typing import List
+
+from mycroft import intent_handler
```
Comment (NeonDaniel) [inline neon_homeassistant_skill/__init__.py]: I think this has a reference in `ovos_workshop` now that should help with dependency warnings (and tests in the future)

File: skill.json [hunk 2]
```diff
@@ -692,7 +692,8 @@
     "systemDeps": false,
     "requirements": {
         "python": [
-            "pfzy==0.3.4"
+            "pfzy==0.3.4",
+            "neon-utils~=1.4.0"
```
Comment (NeonDaniel) [inline skill.json]: I don't think you're using `neon-utils` anywhere here, but you are using `ovos-workshop` and `ovos-utils` (logging)

File: neon_homeassistant_skill/__init__.py [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 from mycroft.messagebus.message import Message
-from mycroft.skills.core import MycroftSkill, intent_handler
 from mycroft.util.log import LOG
```
Comment (NeonDaniel) [inline neon_homeassistant_skill/__init__.py]: This is really coming from `ovos_utils.log`

File: neon_homeassistant_skill/__init__.py [hunk 4]
```diff
@@ -1,12 +1,13 @@
+# pylint: disable=missing-function-docstring,missing-class-docstring,missing-module-docstring
 import asyncio
 from typing import List
+
+from mycroft import intent_handler
 from mycroft.messagebus.message import Message
```
Comment (NeonDaniel) [inline neon_homeassistant_skill/__init__.py]: This is coming from `ovos_bus_client.message`

**Review by NeonDaniel — COMMENTED** [review]
Few comments regarding imports and dependencies

File: skill.json (hunk 2 above)
Comment (mikejgray) [reply skill.json]: I had included neon-utils to fix some issue or other...I'll remove and see what happens

**Pull request merged**